packages/run-protocol/test/amm/constantProduct/test-compareBondingCurves.js

```diff
@@ -39,6 +39,7 @@ const prepareSwapInTest = ({ inputReserve, outputReserve, inputValue }) => {
 
 const testInputGetPrice = (t, inputs, expectedOutput) => {
   const { args, bld } = prepareSwapInTest(inputs);
+  // @ts-ignore
```

Comment:
likewise for the other `@ts-ignore` occurrences

Reply:
done